cart/interfaces/graphql/dto/cartSummary.go

```
@@ -86,3 +86,42 @@ func (cs CartSummary) SumTaxes() *Taxes {
 
 	return &Taxes{Items: taxes}
 }
+
+// SumPaymentSelectionCartSplitValueAmountByMethods – sum
+func (cs CartSummary) SumPaymentSelectionCartSplitValueAmountByMethods(methods []string) *domain.Price {
+	if cs.cart.PaymentSelection == nil {
+		return nil
+	}
+
+	prices := make([]domain.Price, 0, len(methods))
+
+	for qualifier, charge := range cs.cart.PaymentSelection.CartSplit() {
+		found := contains(methods, qualifier.Method)
+		if !found {
+			continue
+		}
+
+		prices = append(prices, charge.Value)
+	}
+
+	if len(prices) == 0 {
+		return nil
+	}
+
+	price, err := domain.SumAll(prices...)
+	if err != nil {
+		return nil
```

do you really want to silence this error?